com.unity.render-pipelines.high-definition/Documentation~/lens-flare-data-driven-asset.md

```diff
@@ -0,0 +1,163 @@
+| Blend Mode              | Select the blend mode of the Lens Flare Element this asset creates:<br />• Additive  <br />• Screen  <br />• Premultiplied <br />• Lerp |
+
+<a name="Transform"></a>
+
+## Transform
+
+![](images/LensFlareTransform.png)
+
+| **Property**            | **Description**                                              |
+| ----------------------- | ------------------------------------------------------------ |
+| Position Offset   | Defines the offset of the lens flare's position in screen space, relative to its source. |
+| Auto Rotate             | Enable this property to automatically rotate the Lens Flare Texture relative to its angle on the screen. Unity uses the **Auto Rotate** angle to override the **Rotation** parameter. <br/><br/> To ensure the Lens Flare can rotate, assign a value greater than 0 to the [**Starting Position**](#AxisTransform)  property. |
+| Rotation                | Rotates the lens flare. This value operates in degrees of rotation. |
+| Size                    | Use this to adjust the scale of this lens flare element. <br/><br/> This property is not available when the [Type](https://github.com/Unity-Technologies/Graphics/pull/3496/files?file-filters[]=.md#Type) is set to [Image](https://github.com/Unity-Technologies/Graphics/pull/3496/files?file-filters[]=.md#Image) and **Preserve Aspect Ratio** is enabled. |
+| Scale                   | The size of this lens flare element in world space.          |
+
+
+<a name="AxisTransform"></a>
+
+## AxisTransform
+
+![](images/LensFlareAxisTransform.png)
+
+| **Property**      | **Description**                                              |
+| ----------------- | ------------------------------------------------------------ |
+| Starting Position | Defines the starting position of the lens flare relative to its source. This value operates in screen space. |
+| Angular Offset    | Controls the angular offset of the lens flare, relative to its current position. This value operates in degrees of rotation. |
+| Translation Scale | Limits the size of the lens flare offset. For example, values of (1, 0) create a horizontal lens flare, and (0, 1) create a vertical lens flare. <br/><br/>You can also use this property to control how quickly the lens flare appears to move. For example, values of (0.5, 0.5) make the lens flare element appear to move at half the speed. |
+
+<a name="Distortion"></a>
+
+## Distortion
+
+![](images/LensFlareRadialDistortion.png)
+
+| **Property**    | **Description**                                              |
+| --------------- | ------------------------------------------------------------ |
+| Enable | True to use or not the radial distortion. |
+| Radial Edge Size | Target size used on the edge of the screen. |
+| Radial Edge Curve | Curve blending from screen center to the edges of the screen. |
+| Relative To Center | If true the distortion is relative to center of the screen otherwise relative to lensFlare source screen position. |
```

Comment:
| Relative To Center | Set this value to **True** to make distortion relative to the center of the screen. Otherwise, distortion is relative to the screen position of the lens flare. |